pkg/terraform/testdata/status-input.yaml

```diff
@@ -0,0 +1,2 @@
+terraform:
+  description: "Status MySQL"
```

Comment:
I know we talked about not using a default directory for the terraform files. That's fine to do (in fact preferred) in another PR.